Common/UnitDefinitions/Density.json

```diff
@@ -193,6 +193,10 @@
     {
       "SingularName": "KilogramPerLiter",
       "PluralName": "KilogramsPerLiter",
+      "BaseUnits": {
+        "L": "Decimeter",
```

Comment:
@tmilnthorp Is decimeter correct? At first I found it strange, but then I realized one liter is decimeter^3 - the same way cubicmeter is meter^3. This base unit stuff is still maturing in my head :-)

Reply:
Yes it is 😃

Reply:
Awesome, please merge and release the nuget if you have the time @tmilnthorp . Or I can do it, just let me know.